portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +14,36 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.*;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    ArrayList<String> Greetings = new ArrayList();
+    ArrayList<String> commentRepo = new ArrayList<String>();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+      
```

Comment:
If deleting code here, please add TODO

Reply:
+1. the good code practice is TODO(Ross): describe what's missing.